css/style.css

```diff
@@ -645,6 +1464,95 @@
     flex-direction: column;
     padding: 11px 0 55px 80px;
   }
+
+  .wrap-information {
+    height: 950px;
+  }
+
+  .info__title {
+    font-size: 62px;
+    text-align: right;
+    padding: 0 70px 30px 0;
+    margin-top: 25px;
+  }
+
+  .red-first, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)